packages/contracts/test/plugins/governance/multisig/multisig.ts

```diff
@@ -258,6 +260,36 @@ describe('Multisig', function () {
       );
       expect(toProtocolVersion).to.not.deep.equal(CURRENT_PROTOCOL_VERSION);
     });
+
+    it('from v1.3.0', async () => {
```

Comment:
wouldn't make sense to test also from v1.0.0 -> v1.4.0 or do we not support this upgrade path?